metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java

```diff
@@ -411,6 +407,18 @@ public Task<DeleteEntityResponse> deleteEntity(@ActionParam(PARAM_URN) @Nonnull
     }, MetricRegistry.name(this.getClass(), "delete"));
   }
 
+  @Action(name = "deleteReferences")
+  @Nonnull
+  @WithSpan
+  public Task<DeleteReferencesResponse> deleteReferencesTo(@ActionParam(PARAM_URN) @Nonnull String urnStr,
```

Comment:
I really like how you broke this out into a separate operation - makes it easy to enable and disable